cloudbuild.yaml

```diff
@@ -1,6 +1,15 @@
 steps:
 - name: 'gcr.io/cloud-builders/docker'
-  args: [ 'build', '-t', 'us.gcr.io/$PROJECT_ID/safe-config-service:$COMMIT_SHA', '.' ]
+  args: [ 'build', '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/charts/safe-config-service:$COMMIT_SHA', '.' ]
+- name: 'us-central1-docker.pkg.dev/clabs-gnosis-safe/cloud-builders-community/helm:3.8.1'
+  args: [ 'package', '-u', 'helm-chart']
+  env:
+  - 'SKIP_CLUSTER_CONFIG=true'
+- name: 'us-central1-docker.pkg.dev/clabs-gnosis-safe/cloud-builders-community/helm:3.8.1'
+  entrypoint: /bin/bash
+  args: ['-c', 'helm push /workspace/safe-config-service-chart-*.tgz oci://us-central1-docker.pkg.dev/$PROJECT_ID/charts']
+  env:
+  - 'SKIP_CLUSTER_CONFIG=true'
 images:
```

Comment:
Don't follow here... the [images](https://cloud.google.com/build/docs/build-config-file-schema#images) field pushes Docker images the docker daemon handles, created with the first cloud build step. The helm chart as a Docker image is generated as a `.tgz` by `helm package` (not really handled by the Docker daemon) so this step pushes the chart in that format.

The Docker image and the Helm chart are Docker images but handled by different tools (docker and helm - i.e. `docker pull `us-central1-docker.pkg.dev\....\chart:version` will return an error, so they are not exactly the same).

Reply:
Should I also remove the `-chart` suffix from the chart image name (`safe-config-service` instead of `safe-config-service-chart`)???

Reply:
Okay I got confused because of the images/charts path confusion earlier. The `images:` section  LGTM now, apologies for the mixup!

> Should I also remove the -chart suffix from the chart image name (safe-config-service instead of safe-config-service-chart)???
Hadn't noticed but yes, I'd be in favour!

Reply:
No worries!!! I think both of us got confused :smile: :smile: 

I will update the PR so that the URLs look like this:
- Container: `us-central1-docker.pkg.dev/$PROJECT_ID/images/safe-config-service:$COMMIT_SHA`
- Helm chart: `us-central1-docker.pkg.dev/$PROJECT_ID/charts/safe-config-service:<CHART_VERSION>`